pkg/deployments/tasks/20210624-stable-pool/test/task.deploy.ts

```diff
@@ -0,0 +1,15 @@
+import hre from 'hardhat';
+import { expect } from 'chai';
+
+import Task from '../../../src/task';
+
+describe('StablePool', function () {
```

Comment:
```suggestion
describe('StablePoolFactory', function () {
```
I'd similarly rename the directory